sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -92,10 +92,16 @@ The specified parameters can also be used to initialize a `ConversationAnalysisO
 
 ```C# Snippet:ConversationAnalysis_AnalyzeConversationWithOptions
 ConversationsProject conversationsProject = new ConversationsProject("Menu", "production");
-AnalyzeConversationOptions options = new AnalyzeConversationOptions(
-    "We'll have 2 plates of seared salmon nigiri.");
+AnalyzeConversationOptions options = new AnalyzeConversationOptions(){
```

Comment:
Repo style is that curly braces are always on their own line.
```suggestion
AnalyzeConversationOptions options = new AnalyzeConversationOptions()
{
```
Of course, change the actual code and then update the snippets.